src/amo/pages/Home/index.js

```diff
@@ -290,7 +282,6 @@ export class HomeBase extends React.Component {
             {_config.get('enableFeatureSponsoredShelf') && isDesktopSite ? (
               <PromotedAddonsCard
                 addonInstallSource={INSTALL_SOURCE_PROMOTED_SHELF}
-                addons={promotedExtensions}
                 loading={loading}
```

Comment:
Maybe `loading` could be moved to `PromotedAddonsCard` too.

Reply:
Good idea!